src/core/urs.ml

```diff
@@ -0,0 +1,95 @@
+(** Soundness of unification rules. *)
```

Comment:
Let's keep the verification of some properties of unification rules for later. (Btw, "urs" is too short.)

Reply:
Yeah, it was just to have the code at hand. Do you want it away from the PR?

Reply:
Yes, I think it is better to do this in another PR.